nebula_examples/package.xml

```diff
@@ -28,6 +28,9 @@
   <depend>velodyne_msgs</depend>
   <depend>yaml-cpp</depend>
 
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
```

Comment:
Packages in Autoware use `autoware_lint_common` here, but I wasn't sure if Nebula can depend on Autoware, @amc-nu thoughts?

Reply:
Sorry I didn’t catch you there
> but I wasn't sure if Nebula can depend on Autoware

Add Autoware as a dependency?

Reply:
I see. Thank you for clarifying.
Yes, the plan is to keep it independent.